packages/cli/src/lib/log.ts

```diff
@@ -1,5 +1,14 @@
 /* eslint-disable no-console */
 
+import {
+  renderInfo,
+  renderWarning,
+  renderFatalError,
+} from '@shopify/cli-kit/node/ui';
+import {BugError} from '@shopify/cli-kit/node/error';
+import {outputContent, outputToken} from '@shopify/cli-kit/node/output';
+import colors from '@shopify/cli-kit/node/colors';
+
 type ConsoleMethod = 'log' | 'warn' | 'error' | 'debug' | 'info';
```

Comment:
table?

Reply:
We are not going to spy on those for now. We'll add them if we ever want to show logs like `[h2:table:<scope>] something`